docs/Q-and-A.md

```diff
@@ -0,0 +1,635 @@
+- [Wallets](#qa-section-wallets)
+- [Signatures](#qa-section-signatures)
+- [Private Key Management](#qa-section-private-key-management)
+- [Blockchain](#qa-section-blockchain)
+- [Agencies](#qa-section-key-agencies)
+- [ION](#qa-section-ION)
+
+# Q&A section General
+
+## *Q: What is Sidetree?
+Sidetree is a protocol for creating a scalable decentralized public key infrastructure ([DPKI](./Glossary.md#public-key-infrastructure)) networks that can run atop of any existing decentralized ledger system (e.g. Bitcoin) and be as open, public, and permissionless as the underlying ledger they utilize.
+
+#### **Q: Why only a protocol, why scalable, why decentralized, why a ledger, why permissionless, why open and public?
+1. Sidetree is a _protocol_ because we need it to be multi applicable. Implementers will be provided with software tools and toolkits and the protocol has both `normative` and non-normative guidelines.
+2. The _scalability_ feature is needed because of the astronomical numbers that are associated with digital identifiers, the numerous life events of each identifier and the mulitple use of each individual identifier. Every individual, group or thing on earth can have multiple identifiers associated with them.
+3. Privacy and guarding control over are the main drivers for the deer wish to _decentralize_ identifiers. If you don’t like being tracked and you don’t like anybody else controlling your identity, decentralization is the way to go. 
+4. A global _ledger_ is needed as a single source of truth.
+5. Permissioned identifiers are limited to a certain domain of application. Sidetree does not want to confine itself and will be as _permissionless_ as the underlaying ledger.
+6. Lastly Sidetree needs to be open and public and at the same the protocol and its applications need to hide and protect all sensitive identity related data. This seems to contradict each other. However in Sidetree it is doable. Cryptography and the protocol safeguard privacy and the _open public_ characteristic offers availability and inclusion at the same time.
+(_@henkvancann_)
+
+## **Q: What is the setting of sidetree?
+The protocol allows users to create _globally unique, user-controlled identifiers_ and manage their associated metadata, all _without the need for centralized authorities_ or trusted third parties. The syntax of the identifier and accompanying data model used by the protocol is conformant to that defined by W3C Decentralized Identifiers. Implementations of the protocol can be codified as their own distinct DID Methods and registered in the W3C DID Method Registry.
+
+## *Q: What is in the name 'Sidetree'?
+The name “Sidetree” combines two hints as to its technical inspirations and superpowers. Each Sidetree network functions as a kind of identity-specific “sidechain” periodically pinned to an underlying chain in much the same way that so-called “Layer 2 Networks”.  These include the Lightning network running atop Bitcoin.\
```

Comment:
should rename all references to chains, ledgers, blockchains to anchoring system.

Reply:
OK, I will do this over the next few days and offer a new PR? Because it needs some additional explanation, because people might expect chains, ledgers and blockchains mentions and I have to explain that we call it an anchoring system and for a good reason.